snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/index/entry/SnomedRelationshipIndexEntry.java

```diff
@@ -430,8 +417,7 @@ public SnomedRelationshipIndexEntry build() {
 	private final boolean destinationNegated;
 
 	private final String destinationId;
-	private final Integer integerValue;
-	private final Double decimalValue;
+	private final BigDecimal numericValue;
```

Comment:
Same here. Please apply the usual backward-compatibility pattern in this document type as well.